### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/codec/issues/28

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,5 +13,8 @@
   "scripts": {
     "test": "tape test/*.js && standard"
   },
+  "engines": {
+    "node": ">=4"
```
Comment (ralphtheninja) [inline package.json:17]: Updating node version in another PR.

File: README.md [hunk 2]
```diff
@@ -99,7 +99,7 @@ To explain this in the grand scheme of things, consider a store like [`leveldown
 
 The `type` string should be a unique name.
 
-## License &amp; copyright
+## License &amp; Copyright
```
Comment (vweevers) [inline README.md]: Or just "License"

File: README.md [hunk 3]
```diff
@@ -10,55 +10,49 @@
 
 ## API
 
-### Codec([opts])
+### `const codec = Codec([opts])`
```
Comment (vweevers) [inline README.md]: Why `const`? Doesn't mean anything here, I think.

File: README.md [hunk 4]
```diff
@@ -10,55 +10,49 @@
...
 
-  Create a new codec, with a global options object.
+Create a new codec, with a global options object.
 
-  This could be something like
+### `codec.encodeKey(key[, opts])`
 
-```js
```
Comment (vweevers) [inline README.md]: We should add an Example section

File: README.md [hunk 5]
```diff
@@ -10,55 +10,49 @@
...
+### `codec.decodeValue(value[, opts])`
 
-  Decode `value` with given `opts`.
+Decode `value` with given `opts`.
 
-### #createStreamDecoder([opts])
+### `codec.createStreamDecoder([opts])`
 
-  Create a function with signature `(key, value)`, that for each key/value pair returned from a levelup read stream returns the decoded value to be emitted.
+Create a function with signature `(key, value)`, that for each key/value pair returned from a levelup read stream returns the decoded value to be emitted.
```
Comment (vweevers) [inline README.md]: `key/value pair` => `key-value pair`

File: README.md (hunk 3 above)
Comment (ralphtheninja) [reply README.md]: No reason.

**Review by vweevers — APPROVED** [review]

**Pull request merged**